app/src/main/java/tech/ula/utils/BusyboxExecutor.kt

```diff
@@ -154,8 +154,7 @@ class BusyboxWrapper {
 
     // Proot scripts expect CWD to be `applicationFilesDir/<filesystem`
     fun addBusyboxAndProot(command: String): List<String> {
-        val commandWithProot = "support/execInProot.sh $command"
-        return listOf("support/busybox", "sh", "-c", commandWithProot)
+        return listOf("support/busybox", "sh", "support/execInProot.sh") + command.split(" ")
```

Comment:
Is splitting the command necessary now?

Reply:
Yes, the `-c` before treated each arg differently.